styles/_mobile.scss

```diff
@@ -29,6 +29,34 @@
     flex-direction: column;
     width: 100%;
 
+    $scroll-bar-height: 7px;
+
+    .highcharts-scrollbar-button {
+        fill: transparent;
+        stroke: transparent;
+        height: $scroll-bar-height;
+    }
+
+    .highcharts-scrollbar-arrow {
+        fill: transparent;
+    }
+
+    .highcharts-scrollbar-thumb {
+        height: $scroll-bar-height;
+        rx: $scroll-bar-height;
+        ry: $scroll-bar-height;
+    }
+
+    .highcharts-scrollbar-track {
+        height: $scroll-bar-height;
+        rx: $scroll-bar-height;
+        ry: $scroll-bar-height;
+    }
+
+    .highcharts-scrollbar-rifles {
+        stroke: transparent;
+    }
+
```

Comment:
@brucebinary Could we move the styles to webtrader-charts instead?

Reply:
I have some considerations for that... but the new scrollbar doesn't look good with the navigator:

![binary_com](https://user-images.githubusercontent.com/32833572/33296373-3c8a395e-d416-11e7-9614-932c46b76d40.png)

Unless webtrader detects mobile view and selectively chooses the style.

Reply:
> Unless webtrader detects mobile view and selectively chooses the style

@brucebinary We can pass "isMobile: true" from next-gen to webtrader-charts.

Reply:
@aminroosta that could work. So in that case I suggest in webtrader-charts add a variable `enableMobileView`. We could also remove the `enableNavigator` setting since enabling mobile view disables the navigator.